src/vs/workbench/contrib/chat/browser/chatEditing/notebook/chatEditingNotebookEditorIntegration.ts

```diff
@@ -415,6 +426,31 @@ class ChatEditingNotebookEditorWidgetIntegration extends Disposable implements I
 							.catch(err => { this.logService.warn(`Error revealing change in view: ${err}`); });
 						return true;
 					}
+					// --- Start Positron ---
+					// For Positron notebooks, cellViewModel may be undefined
+					// Still update the index and try to reveal via cell editor integration
+					// TODO: Make adapter positron notebook cell classes to ICellViewModel so we don't need custom logic here.
```

Comment:
Another option is to extend the types in `IPositronNotebookEditor` and add impls to our cell classes or change our impls to fit theres, if the interface isn't too large/different